### DISCUSSION TITLE
javascript questions?

### DISCUSSION THREAD

**Discussion by chunghow3** [SilentVoid13/Templater#168] [Help] [answered]
hi there! 

i've been tinkering with Templater and am excited at hte possibilities.. managed to get a few syntax going but am having trouble triggering the javascript portions.. wondering if someone could help with the following that is supposed to trigger action on a certain day?

<%* if (tp.date.now ("dd") = ("Th") ) { %>
- [ ] write this down
<%* } %>

**Comment by SilentVoid13** [comment, marked as answer]
Hey @chunghow3, the assertion in your code has an error, here is the fixed version:

```
<%* if (tp.date.now("dd") === "Th") { %>
- [ ] write this down
<%* } %>
```
